app/src/main/AndroidManifest.xml

```diff
@@ -2,18 +2,20 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.sun.android">
 
-    <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-
     <application
         android:allowBackup="true"
         android:icon="@mipmap/icon"
         android:label="@string/name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Structure_Android">
+        <receiver
+            android:name=".bai13_BroadCast.CustomReceiver"
+            android:enabled="true"
+            android:exported="true"></receiver>
```

Comment:
thằng này có cần nữa ko , e đang dùng register broadcast trong activity , nghĩa là e chỉ cho phép nó hoạt động trong nội bộ activity thôi. Còn nếu đăng kí ở đây là hđ trên toàn app rồi. Dùng 1 cách đăng kí thôi nhé